server/app/controllers/admin/Filters.java

```diff
@@ -0,0 +1,7 @@
+package controllers.admin;
+
+/** Class to hold application filter constants shared between different views. */
+public final class Filters {
```

Comment:
suggestion (blocking): i don't think this class will be necessary if query params are specified in the route and method signature for the action